src/components/Home/Search/Search.tsx

```diff
@@ -1,7 +1,48 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import { ENDPOINTS } from '../../../helpers/api';
+import { LocationSearch } from '../../../helpers/Interfaces';
+import { useDebounce } from './../../../hooks/useDebounce';
+import { useGetRequest } from './../../../hooks/useGetRequest';
+import SearchResults from './SearchResults/SearchResults';
+import styles from './Search.module.scss';
 
 const Search: React.FunctionComponent = () => {
-  return <input placeholder="Search for location" />;
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const debouncedSearchTerm: string = useDebounce(searchTerm);
+  const {
+    data: searchResults = { locations: [] },
+    loading = true,
+    error = null
+  }: { data: LocationSearch; loading: boolean; error: string | null } = useGetRequest(
+    ENDPOINTS.locationSearch,
+    debouncedSearchTerm
+  );
+  const [displaySearchResults, setDisplaySearchResults] = useState<boolean>(false);
+
+  const inputChangeHandler = (event: FormEvent<HTMLInputElement>): void => {
+    setDisplaySearchResults(true);
+    setSearchTerm(event.currentTarget.value);
```

Comment:
why not using event.target.value?

Reply:
Because of this 
![screenshot013](https://user-images.githubusercontent.com/96768501/177998770-d19627cb-6f28-47b3-8f25-8f4699c191a6.png)

Is the type correct here? 
`(event: FormEvent<HTMLInputElement>)`

Reply:
for onChange handler you can use `(event: ChangeEvent<HTMLInputElement>)`